scripts/build_symbol_suppression.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# find exports of the form 'PGDLLEXPORT PGFUNCTION_INFO_V1(function_name)'
+EXPORTS=$(grep -F -r 'PGDLLEXPORT' $1 | grep -oP 'PG_FUNCTION_INFO_V1\(\K[^\)]+')
+
+# find exports of the form 'PGDLLEXPORT <type> function_name(...)'
+EXPORTS2=$(grep -roE 'PGDLLEXPORT \w+ \w+' $1 | awk '{print $NF}')
+
+# concatenate the two groups
+EXPORTS="$EXPORTS
+$EXPORTS2"
+
+# remove duplicates
+SYMBOLS=$(echo "$EXPORTS" | sort -u | tr '\n' ';')
+
+# build version script 
+# this script will place everything except undefined symbols and 
+echo "LANTERN {
```

Comment:
What is the significance of "LANTERN" here? is it the version name? the section name in the linker script?

Reply:
It's the version name, I wasn't sure if it made sense to use a specific version